Dockerfile

```diff
@@ -1,4 +1,13 @@
-FROM nvidia/cuda:11.7.0-devel-ubuntu22.04
+# Image Args
+ARG UBUNTU_VER
+ARG CUDA_VER
+
+FROM nvidia/cuda:${CUDA_VER}-devel-ubuntu${UBUNTU_VER}
+ARG TORCH_VER
+ARG TORCHVISION_VER
+ARG TORCHAUDIO_VER
+ARG PYTORCH_EXTRA_IDX_URL
+
 LABEL maintainer="Edgar Y. Walker <eywalker@uw.edu>"
```

Comment:
Could you also add yourself to the maintainer @Synicix ?

Reply:
Sure